### ISSUE TITLE
Script for training condition on text

### ISSUE THREAD

**Issue by trahman8** [SongweiGe/TATS#2]
Thanks for the awesome project. How can I train final model condition on text?

**Comment by SongweiGe** [comment]
We have added the relevant instructions on how to train a conditional transformer and sample videos with it. Please check the ReadMe file and feel free to let me know if you have any questions!

**Comment by trahman8** [comment]
Thanks for uploading the  code.

**Comment by trahman8** [comment]
Hi, 
If I want to load pretrained weight for text to video generation and fine tuned the model for another dataset. I found there are two pretrained weight (1) vqgan and (2) gpt_ckpt. I can use with --vqvae vqgan_ckpt. But how to load gpt_ckpt to load the transformer weight?

**Comment by SongweiGe** [comment]
Hi, if you want to fine tune from a checkpoint, you can use the `--resume_from_checkpoint=your_checkpoint_file` when using any training scripts. If you are using pyotch-lightning > 2.0, this should be `--ckpt_path`. See https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html?highlight=resume_from_checkpoint#resume-from-checkpoint.

Yes, you need both vqgan and gpt ckpts to do generation. They are trained separately. VQGAN is first trained. When training transformer models, VQGAN weights are used but frozen, so only gpt ckpt is saved.

**Comment by trahman8** [comment]
Thanks for the guidelines. Another question to train text-video generation which data-loader did you use? tats/coinrun/coinrun_dataset.py OR tats/coinrun/coinrun_dataset_v2.py. 

I tried to fine-tuned using given pretrained wight, but it generates wired video. Do I need to decrease learning rate when finetune with given weight?

**Comment by SongweiGe** [comment]
We used an old, internal version of the MUGEN dataset. I will be working on the improvements to support the new official dataloader. 

I don't have any experience with the finetuning. How was your loss curve for finetuning?

**Comment by trahman8** [comment]
When I fine-tune using new data loader, the loss is decreasing and acc1 and acc5 is quite good during training. But during inference, the generated video become worst.

**Comment by Crane-YU** [comment]
@trahman8 Hi, have u tried to change `text_seq_len`?
